jina/drivers/__init__.py

```diff
@@ -170,18 +170,20 @@ def __getstate__(self) -> Dict[str, Any]:
 
 class BaseRecursiveDriver(BaseDriver):
 
-    def __init__(self, depth_range: Tuple[int] = (0, 0), apply_order: str = 'post',
+    def __init__(self, depth_range: Tuple[int] = (0, 1), apply_order: str = 'post',
```

Comment:
revert it back

Reply:
fixed. Add notes for the failed tests on #814